src/CookieUtil.php

```
@@ -0,0 +1,40 @@
+<?php
+
+namespace Http\Message;
+
+final class CookieUtil
+{
+    /**
+     * Handles dates as defined by RFC 2616 section 3.3.1, and also some other
+     * non-standard, but common formats.
+     *
+     * @var array
+     */
+    private static $dateFormats = array(
+        'D, d M y H:i:s T',
+        'D, d M Y H:i:s T',
+        'D, d-M-y H:i:s T',
+        'D, d-M-Y H:i:s T',
+        'D, d-m-y H:i:s T',
+        'D, d-m-Y H:i:s T',
+        'D M j G:i:s Y',
+        'D M d H:i:s Y T',
+    );
+
+    /**
+     * @see https://github.com/symfony/symfony/blob/master/src/Symfony/Component/BrowserKit/Cookie.php
+     *
+     * @param string $dateValue
+     * @return \DateTime|false
```

Should I rather throw an exception ?